src/XMakeBuildEngine/BackEnd/Components/Communications/NodeProviderOutOfProc.cs

```diff
@@ -112,14 +112,12 @@ public bool CreateNode(int nodeId, INodePacketFactory factory, NodeConfiguration
             }
 
             // Start the new process.  We pass in a node mode with a node number of 1, to indicate that we 
-            // want to start up just a standard MSBuild out-of-proc node. 
-            string commandLineArgs = " /nologo /nodemode:1 ";
-
-            // Disable node re-use if it is not requested (because no argument means enable reuse).
-            if (!ComponentHost.BuildParameters.EnableNodeReuse)
-            {
-                commandLineArgs += "/nodeReuse:false";
-            }
+            // want to start up just a standard MSBuild out-of-proc node.
+            // Note: We need to always pass /nodeReuse to ensure the machine-wide msbuild.rsp setting for 
```

Comment:
I'd just say "any response file setting for"--we really mean "the one next to msbuild.exe" and the meaning of that changes--for a build from GitHub it's just in the local folder.